interface/typed_signature.py

```diff
@@ -20,6 +20,8 @@ def __init__(self, obj):
         self._type = type(obj)
         if isinstance(obj, (classmethod, staticmethod)):
             self._signature = signature(obj.__func__)
+        elif isinstance(obj, property):
```

Comment:
Using `fget` works well. Just pushed a commit with that change. (Also removed check in `interface.py`)